packages/build-tools/src/builders/android.ts

```diff
@@ -300,7 +313,7 @@ async function generateCacheKeyAsync(workingDirectory: string): Promise<string>
 
   try {
     const key = await hashFiles([lockPath]);
-    return `android-ccache-${key}`;
+    return `${CACHE_KEY_PREFIX}-${key}`;
```

Comment:
```suggestion
    return `${CACHE_KEY_PREFIX}${key}`;
```

Same changes for ios